### DISCUSSION TITLE
Device compatibility issue

### DISCUSSION THREAD

**Discussion by rez1Pro** [aniruddha-adhikary/mrt-buddy#111] [General]
Android V14 , Redmi series .. Not compatible but NFC exists in my phone. please try to solve it

**Comment by aniruddha-adhikary** [comment]
@rez1Pro Thank you for reporting this issue. To troubleshoot, please follow these steps:

1. Install the NXP TagInfo app from the Google Play Store: [TagInfo App](https://play.google.com/store/apps/details?id=com.nxp.taginfolite&hl=en).  
2. Use the app to scan your MRT/Rapid Pass card. If your phone vibrates or displays information in the app, the NFC hardware is working as expected.  
3. If nothing happens, this may indicate an issue with your phone’s NFC hardware compatibility.

Please note that some NFC devices do not support Class F NFC, which is required for our MRT system. This might explain the incompatibility.

Additionally, try downloading the APK directly from our latest release: [MRT Buddy v0.0.21](https://github.com/aniruddha-adhikary/mrt-buddy/releases/tag/v0.0.21). Install it and check if it resolves your issue. (Without Play Store)

Let us know if you need further assistance! 😊 Do let us know what happens!
